hadoop-hdds/framework/src/main/java/org/apache/hadoop/hdds/utils/HAUtils.java

```diff
@@ -413,47 +418,44 @@ public static File getMetaDir(DBDefinition definition,
     return caCertPemList;
   }
 
+  /**
+   * Retry for ever until CA list matches expected count.
+   * @param task - task to get CA list.
+   * @return CA list.
+   * @throws IOException
```

Comment:
```suggestion
```

Reply:
Why is that, it throws exception right?

Reply:
>  it throws exception right?

Yes, but `@throws` also needs a description, to document when/why the exception is thrown.  The fact that it is thrown is already documented by the `throws` part of the method declaration.